frontend/src/routes.js

```diff
@@ -107,4 +108,27 @@ export const routes = {
       component: UserProfile,
     },
   },
+
+  userDashboard: {
+    route: {
+      exact,
+      path: "/users/:userId/dashboard",
+      component: Dashboard,
+    },
+  },
+
+  dashboard: {
+      route: {
+        exact,
+        path: "/dashboard",
+        component: Dashboard,
+      },
+      sliderNavigation: {
```

Comment:
please remove from the slider navigation.

We dont actually want this to show up for the users until we are ready. If someeone clicks on thi link and get to a page that just says "TODO" then that's not a good user experience.

For now we need to be able to navigate to it just by using the url bar. We aren't ready to show it to anyone yet.